src/test_l1_dsDisplay.c

```diff
@@ -112,6 +112,7 @@ static bool extendedEnumsSupported=false;
    }\
```

Comment:
remove #include "dsVideoPortSettings.h".
Also remove DS_ASSERT_AUTO_TERM_NUMERICAL and DS_ASSERT_AUTO_TERM_STRING if not used

Reply:
done